ts/parts-map/MapSeries.ts

```diff
@@ -1317,7 +1317,7 @@ seriesType<Highcharts.MapSeriesOptions>(
                     (series.mapMap as any)[(point as any)[joinBy[1]]];
                 if (mapPoint) {
                     // This applies only to bubbles
-                    if ((series as Highcharts.MapBubbleSeries).xyFromShape) {
+                    if ((series as any).xyFromShape) {
```

Comment:
Why is it necessary to cast to `any` in this case?

Reply:
Because `MapSeries` can never become `MapBubbleSeries`, because `MapBubbleSeries` is based on `BubbleSeries`. I wonder, if this gets actually ever triggered.

Reply:
Before I merge the PR, can you confirm that this is correct @oysteinmoseng?

Reply:
@bre1470 I had a chat with @oysteinmoseng and we found that this condition is triggered if the following gets called: https://github.com/highcharts/highcharts/blob/2a5bfe812ffbdd2e1f808bdfe67cfac6a6cb558a/ts/parts-map/MapBubbleSeries.ts#L249-L252

I am not sure how to solve the types to handle this case.

Reply:
So it is misuse of mixin from another series prototype. We will have to solve this when removing the any cast.

Reply:
Okay, then we'll leave the `any` for now.